examples/code_review_agent/skills/code-review/scripts/run_go_vet.sh

```
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -euo pipefail
+
+PACKAGE_PATH="${1:-.}"
+
+mkdir -p out
+
+echo "=== go vet ===" > out/go_vet_results.txt
+
+if command -v go > /dev/null 2>&1; then
+    go vet "$PACKAGE_PATH" 2>&1 >> out/go_vet_results.txt || true
```

Shellcheck (0.11.0)

[warning] To redirect stdout+stderr, 2>&1 must be last (or use '{ cmd > file; } 2>&1' to clarify).

(SC2069)